src/components/Header/index.tsx

```diff
@@ -0,0 +1,76 @@
+import { useState } from 'react';
+
+import Link from 'next/link';
+
+import IconArrow from 'assets/icons/icon-arrow.svg';
+import IconLogo from 'assets/icons/logo.svg';
+
+const categories = [
+  { name: 'All', href: '/' },
+  { name: 'Web', href: '/tags/web' },
+  { name: 'Mobile', href: '/tags/mobile' },
+  { name: 'Design', href: '/tags/design' },
+  { name: 'Product', href: '/tags/product' },
+];
+
+const Header = () => {
+  const [isMenuOpen, setIsMenuOpen] = useState(false);
+
+  return (
+    <header className="app-header">
+      <nav
+        className={`app-navigation-bar navbar ${
+          isMenuOpen ? 'app-navigation-bar--menu-open' : ''
+        }`}
```

Comment:
maybe we could simplify it using `classNames`?

```suggestion
        className={classNames('app-navigation-bar navbar',  { 'app-navigation-bar--menu-open' : isMenuOpen })}
```